src/types.ts

```diff
@@ -57,3 +57,12 @@ export interface Waypoint {
   lat: number;
   snapped: boolean;
 }
+
+export type Mode =
+  | "edit-attribute"
+  | "edit-geometry"
+  | "route"
+  | "point"
+  | "free-polygon"
+  | "snap-polygon"
+  | "split-route";
```

Comment:
Aha, `npm run check` to the rescue!

```
StreetViewMode.svelte:103:6                   
Error: This comparison appears to be unintentional because the types 'Mode' and '"street-view"' have no overlap. (ts)

{#if $currentMode == thisMode}
```

We need `street-view` here; it was added during a rebase

Reply:
Ahhh I was seeing this error and looking in the wrong types file: was very confused why I couldn't see the problem!

A great example of why I should've deleted the duplicate types file